crypto/signer.go

```diff
@@ -35,10 +35,20 @@ var sigpbHashLookup = map[crypto.Hash]sigpb.DigitallySigned_HashAlgorithm{
 // Signer is responsible for signing log-related data and producing the appropriate
 // application specific signature objects.
 type Signer struct {
+	KeyID  int64
 	Hash   crypto.Hash
 	Signer crypto.Signer
 }
 
+// NewSigner returns a new annotated signer.
```

Comment:
Needs a comment to explain what "annotated" means.

Reply:
done

Reply:
Done where?

Reply:
Must have gotten lost. Fixed comment again.